### DISCUSSION TITLE
Help with Prometheus

### DISCUSSION THREAD

**Discussion by CooperDev-Code** [postalserver/postal#3126] [Help with using Postal]
Has anyone had any sucsess useing prometheus to scrap the /metrics, not to sure what im doing wrong.

**Comment by willpower232** [comment]
Have you seen https://docs.postalserver.io/features/health-metrics ?

Presumably you would need to make sure those ports were available in order to reach /metrics

**Reply by CooperDev-Code** [reply]
Yeah, I've tired setting different ports, disabled firewall, just can't seem to connect to it and don't know what I'm doing wrong.

Any help or pointers would be appreciated.

**Reply by willpower232** [reply]
Can you see the ports being used with `netstat -lnp`?
